docs/content/config/advanced/tips/server-multiple-ip-addresses.md

```diff
@@ -0,0 +1,43 @@
+---
+title: 'Use Cases | Binding DMS to a specific network'
+hide:
+  - toc
+---
+
+!!! warning "Advice not extensively tested"
+
+    This configuration has only been used with `network: host`, where you have direct access to the host interfaces.
+
+If your host system is running multiple IPv4 and IPv6 IP-addresses, you probably have an interest to
+bind outgoing SMTP connections to specific IP-addresses to ensure MX records are aligned with
+PTR-records in DNS when sending emails to avoid getting blocked by SPF for example.
```

Comment:
Maybe the phrase about SPF should be more clear:

bind outgoing SMTP connections to specific IP-addresses to ensure MX records are aligned with PTR and SPF records in DNS when sending (and receiving emails).

You expect at least the resolving IP-addresses from the MX records in DNS to have reverse DNS (PTR records) matching the MX records. 
And those IP-addresses for sending emails need to be specified in the SPF records in DNS. (should be correct for bounces etc as well, so affects receiving as well?) 

Agree @polarathene ?

Reply:
I don't think the PTR records have to match MX records since that's only related to where mail should be delivered/received for that domain.

IIRC, when this type of check is done, it's not necessarily strict, it's usually acceptable to resolve the sending mail-server shares the same IP as the PTR record and the address it points to are the same.

Example setup:
- DMS server + config:
  - PTR `1.2.3.4` => `example.com`
  - A `example.com` => `1.2.3.4`
  - A `mail.example.com` => `1.2.3.4`
  - `compose.yaml` with hostname `mail.example.com` + `setup email add hello@example.org`
- DNS support for mail account `hello@example.org` (_mail accounts don't need to share the same domain or server as DMS_):
  - A `example.org` => `5.6.7.8` (_not important to DMS_)
  - MX `example.org` => `mail.example.com` (_Mail sent to `example.org` will be handled by DMS, not relevant to example below_)
  - TXT (SPF) `example.org` =>`"v=spf1 mx ip4:1.2.3.4 a:mail.example.com -all"` (_only the `mx` should be necessary, but could alternatively specify an IP address authorized to send mail on behalf of `example.org`, that IP address could implicitly be resolved for an `A` record lookup too if the `MX` of `example.org` is not appropriate_)

MTA connection example:
1.When connecting to another MTA to send mail to (_with sender `hello@example.org`_), DMS will greet the MTA with HELO / EHLO of `mail.example.com`.
2. The MTA we are sending the mail to (_eg: belonging to a recipient, or is a relay service_) may do a PTR lookup for the IP that `mail.example.com` points to (`1.2.3.4`), which the PTR record resolves to `example.com`, which in turn resolves back to `1.2.3.4`, same as DMS at `mail.example.com`, pass ✔️ 
3. SPF record can also be checked, verifying that `example.org` authorizes `mail.example.com` (_or in this case anything sending from that IP_).

I may be mistaken if I'm recalling the above incorrectly. But the `PTR` check is tied to DMS hostname (_that Postfix gets configured for_), like with the TLS certificate, the trust is in the mail-server here, the sender address is not relevant (_separately checked via DNS lookup for records like SPF that authorize DMS server to send_).

Reply:
So perhaps it should be advising in the docs here to match the IP that `hostname` (`compose.yaml`) would resolve to via DNS?